soroban-sdk/src/testutils/arbitrary.rs

```diff
@@ -876,7 +869,7 @@ mod composite {
         String(<Vec<String> as SorobanArbitrary>::Prototype),
         Symbol(<Vec<Symbol> as SorobanArbitrary>::Prototype),
         Vec(<Vec<Vec<u32>> as SorobanArbitrary>::Prototype),
-        Map(<Map<u32, u32> as SorobanArbitrary>::Prototype),
+        Map(<Vec<Map<u32, u32>> as SorobanArbitrary>::Prototype),
```

Comment:
Why does the Map contain a Vec or Maps?

Reply:
fwiw i recognize this is a lot of code just to produce random `Val`s, and there might be more clever and thorough ways to do it